samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/api/fake_api.dart

```diff
@@ -126,7 +128,9 @@ class FakeApi {
         List<String> contentTypes = ["application/json"];
 
 
-            var serializedBody = _serializers.serialize(body);
+
+            var bodySerializer = _serializers.serializerForType(bool);
```

Comment:
Does this work or does this need extra handling for primitives?

Reply:
After looking at the StandardJsonPlugin, I think this needs special handling for primitives. Probably a plain jsonEncode without any serializer?